packages/fx-core/src/plugins/resource/blazor/constants.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+
+export class Constants {
+  static readonly emptyString = "";
+}
+
+export class AzureInfo {
+  static readonly webappNameLenMax = 24;
+  static readonly suffixLenMax = 12;
+  static readonly zipDeployURL = (appName: string) =>
+    `https://${appName}.scm.azurewebsites.net/api/zipdeploy`;
+  // TODO: static readonly requiredResourceProviders = ["Microsoft.Web"];
+  static readonly aadMetadataAddress = (tenantId: string) =>
+    `https://login.microsoftonline.com/${tenantId}/v2.0/.well-known/openid-configuration`;
+}
+
+export class RegularExpr {
+  static readonly allCharToBeSkippedInName: RegExp = /[^a-zA-Z0-9]/g;
+}
+
+export class DefaultProvisionConfigs {
+  static readonly appServicePlansConfig = (location: string) => ({
+    location: location,
+    kind: "app",
+    sku: {
+      name: "B1",
+    },
+  });
+
+  static readonly webAppConfig = (location: string) => ({
+    kind: "app",
+    location: location,
+    sku: {
+      name: "B1",
+    },
+  });
+}
+
+export class BlazorPluginInfo {
+  static readonly pluginName = "fx-resource-blazor";
+  static readonly displayName = "Blazor";
+  static readonly alias = "bz";
+  static readonly issueLink = "https://github.com/OfficeDev/TeamsFx/issues/new";
```

Comment:
Since the issue link is the same across all plugin, I am not sure if it is filled by solution/telemetry helper level. Can you help check if we need to manually set this value in our plugin?

Reply:
Seems it's filled by extension. Let me remove it.